lib/rules/avoid-using-needs-in-controllers.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const ember = require('../utils/ember');
+
+//------------------------------------------------------------------------------
+// Ember object rule - Avoid using needs in controllers
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Avoids using needs in controllers',
+      category: 'Ember Object',
+      recommended: true
```

Comment:
I think it should be `false` until we actually add it to the recommended config.